scenarios/krvtz/postfix.yaml

```diff
@@ -0,0 +1,15 @@
+# postfix SASL bruteforcing
```

Comment:
This file shouldnt be needed if the failed `spam-attempt` matches the original postfix scenarios